### DISCUSSION TITLE
Where is the Library View in CarPlay?

### DISCUSSION THREAD

**Discussion by dkBrazz** [TortugaPower/BookPlayer#813] [Q&A] [answered]
As far as I remember there was a Library View in CarPlay previously, and I could switch between "library" and "recent", now I can only see Recents View which is not inconvenient in many cases
Is it a conscious decision to remove the library or a bug?

**Comment by GianniCarlo** [comment, marked as answer]
Hi @dkBrazz ! when rewriting the CarPlay implementation to use the new apis, I noticed that there are no events to know when one is approaching the last element of the list, this would mean that we would have to load all the items at the library root at once, which is not ideal. I went with only the Recent list to see if that was enough for most use cases (since we cap that at 25 items I think), but we've heard back from users, and it's on our backlog to add back

**Reply by dkBrazz** [reply]
Got it, thanks for the explanation!
